DependencyInjection/Configuration.php

```diff
@@ -54,6 +54,10 @@ public function getConfigTreeBuilder()
                     ->info('Send data user on each master request (useful if you do not force users to disconnect when setting up the bundle or add new properties in their profile. WARNING: preferred used @Mixpanel\UpdateUser() or @Mixpanel\Set() at connection for performances)')
                     ->defaultValue($this->debug)
                 ->end()
+                ->booleanNode('user_data_display_errors')
```

Comment:
Can we name this field as: `throw_on_user_data_attribute_failure`? It's more verbose but conveys better what it refers to.